migration-guide.md

```diff
@@ -0,0 +1,72 @@
+
+
+## Existing features with new configuration options
+
+This section unpacks the small changes in Edge from the Proxy. These are ports of existing features or configuration that have small changes. These shouldn't affect how you use Edge and the ideas here are similar to the Proxy, only small details have changed.
+
+- Unleash URL. The Proxy requires that you specify an Unleash URL to the upstream server, in the format https://{unleashUrl}/api. Edge has changed this, the URL that Edge requires is https://{unleashUrl}, without the `/api` suffix.
+
+- Backend SDK support. The Proxy does support connecting to backend SDKs, but it requires some configuration and setting some experimental feature flags. Edge supports this out the box, no additional configuration is required.
+
+
+## New features
+
+- Offline mode. Edge can be used in a purely offline mode. While the Proxy does support this, it requires a little work to get there. Edge, on the other hand, supports this directly and out the box. This can be useful for testing or development environments where you don't want to have to run an Unleash instance or want to freeze your configuration in place. Example run command:
+
+    ``` sh
+    ./edge offline --tokens "*.development.some-test-token"  --bootstrap-file ./examples/features.json
+    ```
+
+    This will start Edge in offline mode and set the initial feature set to the contents of the `features.json` file. Edge will not send metrics to upstream Unleash instances, update its feature information or dynamically resolve tokens. Note that you must set a token or tokens on startup in this mode - Edge will only use this set of tokens to validate incoming requests, this doesn't have to be a valid Unleash token, so this is very similar to the original Proxy tokens.
+
+- Daisy chaining. Edge is capable of both mimicking an Unleash server and talking to an Unleash server. This means that Edge can use another Edge instance as another upstream source. This can be extremely useful for high scaling scenarios where you can have a single instance close to your SDKs that retrieves all toggles for all projects and environments and specific Edge instances syncing to the parent Edge instance, with much more narrowly defined API tokens for security reasons. No extra configuration is needed to achieve this, simply point the downstream Edge instance at the upstream Edge instance and it will automatically resolve the tokens and features from the upstream Edge instance. Metrics will propagate up the daisy chain from downstream Edge instances to upstream instances until the metrics hit the end of the chain in either an offline Edge instance or an upstream Unleash instance. Note that if the daisy chain terminates in an offline Edge instance, the metrics will be discarded.
+
+
```

Comment:
Silly thing, but do we need three blank lines at the end of the file? Isn't one enough?

```suggestion


```

Reply:
It's more than enough, this is just me apparently being over eager with the enter key